proto/workflow.proto

```diff
@@ -134,3 +134,35 @@ message GetReposResponse {
   // Repos fetched from the provider.
   repeated Repo repo = 2;
 }
+
+// Metadata about a workflow that is sent as part of the build metadata BEP
+// event.
+message WorkflowActionMetadata {
+  // Action name as defined in `buildbuddy.yaml`.
+  //
+  // Example: "Test all targets"
```

Comment:
I think this is maybe editor-specific but the way these comments look is confusing. I would say they are malformed.

There's an unnecessary newline between the top level comment "Action name as defined in `buildbuddy.yaml`." and the example "Example: "Test all targets".

There is no newline between the field itself and the beginning of the next comment, and there should be.

Reply:
Added spaces between the fields & beginning of the following comment.

As for the blankline before the example, the intent is to make the example a new paragraph. My mental model is that the way to do a new paragraph is to have two newlines (i.e. introduce a blank line), otherwise there is no way to differentiate between a new paragraph and a continuation of the same paragraph. clang-format uses this model as well in its comment formatting, and (somewhat annoyingly) this has the effect that it treats the example as part of the preceding paragraph when re-wrapping lines, and sometimes will put the example on the previous line.

Anyway, removed the blankline for consistency with the other comments -- just something to consider.